jwst/refpix/reference_pixels.py

```diff
@@ -181,6 +184,56 @@ def sigma_clip(self, data, dq, low=3.0, high=3.0):
             mean = data[goodpixels].mean(dtype=np.float64)
         return mean
 
+    def transfer_to_model(self, input_model):
+        """
+        Transfer the reference-pixel corrected data arrays from the
+        dataset to the input model
+
+        Parameters:
+        -----------
+
+        input_model: JWST datamodel
+
+        The datamodel the data arrays are to be transferred to
```

Comment:
"The datamodel to which the data arrays are to be transferred"  (I find it ironic that an Amurican is correcting a Brit's grammar ;-) )